src/types.ts

```diff
@@ -2,6 +2,19 @@ import type { ReactNode } from 'react';
 import type { Address, Chain } from 'viem';
 import type { EASSchemaUid } from './identity/types';
 
+/**
+ * Note: exported as public Type
+ */
+export type AppConfig = {
+  appearance?: {
+    name?: string | null; // The name of your application
+    logo?: string | null; // The URL of your application logo
+    mode?: Mode | null; // Optionally determines color scheme based on OS preference or user selection
+    theme?: ComponentTheme | null; // Optionally sets the visual style for components
```

Comment:
Nit: Add optional prefix in description for name and logo, similar to the other params